files/en-us/web/api/mouseevent/initmouseevent/index.html

```diff
@@ -120,7 +122,9 @@ <h3 id="Result">Result</h3>
 
 <h2 id="Specifications">Specifications</h2>
 
-{{Specifications}}
+<p>This feature is not part of any specification. It is no longer on the track of becoming a standard.</p>
```

Comment:
```suggestion
<p>This feature is not part of any specification. It is no longer on track to becoming a standard.</p>
```
I think this is fine the way it is, but “on track to becoming a standard” seems the most idiomatic/typical way of expressing this.

Reply:
I will adopt it for everywhere else :-)